ldap/config.go

```diff
@@ -29,6 +29,14 @@ const (
 
 	// DefaultTLSMaxVersion for the ClientConfig.TLSMaxVersion
 	DefaultTLSMaxVersion = "tls12"
+
+	// DefaultOpenLDAPUserPasswordAttribute defines the attribute name for the
+	// openLDAP default password attribute which will always be excluded f
+	DefaultOpenLDAPUserPasswordAttribute = "userPassword"
+
+	// DefaultADUserPasswordAttribute defines the attribute name for the
+	// AD default password attribute
```

Comment:
Might want to add "which will always be excluded" similar to above?

Reply:
ty!